app/components/post_body/index.js

```diff
@@ -52,6 +57,20 @@ function mapStateToProps(state, ownProps) {
     const isUserCanManageMembers = canManageChannelMembers(state);
     const isEphemeralPost = isPostEphemeral(post);
 
+    const {config, license} = state.entities.general;
```

Comment:
We have selectors for these, like 
``import {getConfig, getLicense} from 'mattermost-redux/selectors/entities/general';``

Reply:
I have switched all instances of direct access of `license` and `config` from state to use selectors.